src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolume.cxx

```diff
@@ -823,8 +765,8 @@ const char* vtkPlusPasteSliceIntoVolume::GetCompoundingModeAsString( Compounding
     return "LATEST";
   case MEAN_COMPOUNDING_MODE:
     return "MEAN";
-  case IMPORTANCE_MASK_COMPOUNDING_MODE:
-	  return "IMPORTANCEMASK";
+  case IMPORTANCE_MASK_COMPOUNDING_MODE:
+	return "IMPORTANCEMASK";

```

Comment:
I think this is a tab instead of spaces? I will update our hook to check for this.